build/CopyFilesToStagingDir.ps1

```diff
@@ -35,27 +35,27 @@ function PublishFile {
 
 PublishFile $OverrideDir\DynamicDependency-Override.json $FullPublishDir\
 
-PublishFile $FullBuildOutput\ProjectReunion_DLL\Microsoft.ProjectReunion.dll $FullPublishDir\Microsoft.ProjectReunion\
-PublishFile $FullBuildOutput\ProjectReunion_DLL\Microsoft.ProjectReunion.lib $FullPublishDir\Microsoft.ProjectReunion\
-PublishFile $FullBuildOutput\ProjectReunion_DLL\Microsoft.Windows.AppLifecycle.winmd $FullPublishDir\Microsoft.ProjectReunion\
-PublishFile $FullBuildOutput\ProjectReunion_DLL\Microsoft.Windows.PushNotifications.winmd $FullPublishDir\Microsoft.ProjectReunion\
-PublishFile $FullBuildOutput\ProjectReunion_DLL\Microsoft.Windows.ApplicationModel.DynamicDependency.winmd $FullPublishDir\Microsoft.ProjectReunion\
-PublishFile $FullBuildOutput\ProjectReunion_DLL\MsixDynamicDependency.h $FullPublishDir\Microsoft.ProjectReunion\
-PublishFile $FullBuildOutput\ProjectReunion_DLL\wil_msixdynamicdependency.h $FullPublishDir\Microsoft.ProjectReunion\
+PublishFile $FullBuildOutput\WindowsAppSDK_DLL\Microsoft.WindowsAppSDK.dll $FullPublishDir\Microsoft.WindowsAppSDK\
```

Comment:
Since this is product not dev tooling, it might be better to call the DLL WindowsApp.dll.

Reply:
Sounds good to me. Can we make the call? @andrewleader do you concur? Do we need anyone else to weigh in?

Reply:
We should probably use the same name as what appears in the Uninstall Apps/etc. Which I think our two candidate names are Windows App Framework and Windows App Runtime. I'll start an email thread to see if we can get closure on that decision early next week.

Reply:
OK. I'll stick with the name as is for this PR and we can rename it later to the final name. That'll only affect the dll filename, the .vcxproj and references in other .vcxproj so easy enough to do in a follow up PR